app/configs/crashesListViewTable.ts

```diff
@@ -1,14 +1,10 @@
 import {
-  getStartOfYearDate,
   getYearsAgoDate,
   makeDateFilters,
 } from "@/components/TableDateSelector";
-import { crashesListViewColumns } from "@/configs/crashesListViewColumns";
-import { QueryConfig, FilterGroup } from "@/utils/queryBuilder";
+import { QueryConfig, FilterGroup } from "@/types/queryBuilder";
 import { DEFAULT_QUERY_LIMIT } from "@/utils/constants";
 
-const columns = crashesListViewColumns.map((col) => String(col.path));
```

Comment:
this PR removes the array of column names out of the `queryConfig` object and instead let's uses the column definitions that are passed into the `TableWrapper` component.